src/Integration.UnitTests/SonarAnalyzer/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -23,6 +22,7 @@ public class ActiveSolutionBoundTrackerTests
         private SolutionMock solutionMock;
         private ConfigurableActiveSolutionTracker activeSolutionTracker;
         private ConfigurableHost host;
+        private ConfigurableErrorListInfoBarController errorListController;
```

Comment:
ErrorListInfoBarController will be used from the ActiveSolutionBoundTracker service since it already tracks solution evens. If needed we can integrate the controller elsewhere but for the mean while this seemed like a good place to me.